### DISCUSSION TITLE
KTOR SUPPORT

### DISCUSSION THREAD

**Discussion by JabezNzomo99** [Netflix/dgs-framework#178] [Q&A]
Does dgs-framework have support for [Ktor](http://ktor.io/) currently? If yes, is there any sample for integrating with Ktor?

**Comment by paulbakker** [comment]
The DGS framework is designed for Spring - it heavily relies on its dependency injection. I have very limited knowledge of KTOR, so correct me if I’m wrong, but I think KTOR is not integrating with Spring.

Similar questions have come up for Jakarta EE and Micronaut, but we’re not planning to support any non-Spring environments.
